doc/guide/scripting.rst

```diff
@@ -234,3 +236,17 @@ On windows, this example can be called from the cmd prompt using sasview as
 as the python interpreter::
 
     SasViewCom example/cylinder_eval.py
```

Comment:
I don't know if it will be `SasViewCom` or `SasView`. Somebody building the windows exe will have to see if two are needed. Note that the bumps module needs the `-m` as in `SasView -m bumps.cli model.py` but a simple script file does not. This is similar to the python command line interface.

Reply:
Resolving this is waiting on a Windows review of https://github.com/SasView/sasview/pull/2280